internal/js-dependency-scanner/src/main/java/org/wso2/security/tools/scanner/dependency/js/constants/JSScannerConstants.java

```diff
@@ -0,0 +1,99 @@
+    public static final String DAS = "wso2das";
+    public static final String OB = "wso2-ob-solution";
+
+    //constants related to config files
+    public static final String GIT_CONFIG_FILE = "githubconfig.properties";
+    public static final String GIT_ACCESS_TOKEN_CONFIG_FILE = "gitaccesstoken.properties";
+    public static final String ISSUECREATOR_CONFIG_FILE = "issuecreatorconfig.properties";
+    public static final String PRODUCT_LIST_FILE = "productlist.properties";
+    public static final String APIM_CONFIG_FILE = "apimconfig.properties";
+    public static final String EI_CONFIG_FILE = "eiconfig.properties";
+    public static final String DAS_CONFIG_FILE = "dasconfig.properties";
+    public static final String SP_CONFIG_FILE = "spconfig.properties";
+    public static final String IS_CONFIG_FILE = "iamconfig.properties";
+    public static final String JIRA_TICKET_INFO_FILE = "jiraticketinfo.properties";
+    public static final String OB_CONFIG_FILE = "obconfig.properties";
+    public static final String PROPERTIES_FILE_DELIMETER = ",";
+
+    //Config files attributes
+    public static final String PRODUCTS = "products";
+    public static final String ACCESSTOKEN = "accessToken";
+    public static final String USERNAME = "userName";
+    public static final String PASSWORD = "password";
+    public static final String GIT_REPO_NAME = "gitRepoName";
+    public static final String INPUT_SOURCE_TYPE = "inputSourceType";
+    public static final String VERSION_TAG_KEY_WORD = "componentVersionTag";
+    public static final String COMPONENT_REPO = "componentRepo";
+
+    //constants related to git api url
+    public static final String GIT_BASE_URL = "https://api.github.com/repos/wso2/";
+    public static final String GIT_SEARCH_URL = "https://api.github.com/search/code?q=filename:";
+    public static final String POM = "pom.xml";
+    public static final String PACKAGE_JSON = "package.json";
+    public static final String RELEASES = "/releases";
+    public static final String RAW_GIT_BASE_URL = "https://raw.githubusercontent.com/";
+    public static final String ASSETS = "assets";
+    public static final String NAME = "name";
+    public static final String DOWNLOAD_URL = "browser_download_url";
+    public static final String ATUWA_BASE_URL = "http://atuwa.private.wso2.com/WSO2-Products/open-banking-solutions/";
+    public static final String OB_INDEX_FILE = "open-banking-solutions-packs.index";
+    //constants related to directory
+    public static final String PRODUCT_HOME = System.getProperty("user.home") + "/products";
+    public static final String SECURITY_ARTIFACT_HOME = System.getProperty("user.home") + "/securityArtifact";
+    public static final String WEEKLY_RELEASE = "weeklyRelease";
+    public static final String GA_RELEASE = "GARelease";
+    public static final String SECURITY_ARTIFACT_REPO = "https://github.com/wso2/security-artifacts.git";
```

Comment:
Why isn't this read from a config file? And this is not a public URL

Reply:
Resolved